cmd/crowdsec/crowdsec.go

```diff
@@ -134,30 +159,10 @@ func waitOnTomb() {
 				So : we are waiting for the number of buckets to stop decreasing before returning. "how long" we should wait is a bit of the trick question,
 				as some operations (ie. reverse dns or such in post-overflow) can take some time :)
 			*/
-			for {
-				currBucketCount := leaky.LeakyRoutineCount
 
-				if currBucketCount == 0 {
-					/*no bucket to wait on*/
-					break
-				}
-				if currBucketCount != bucketCount {
-					if rounds == 0 || rounds%2 == 0 {
-						log.Printf("Still %d live LeakRoutines, waiting (was %d)", currBucketCount, bucketCount)
-					}
-					bucketCount = currBucketCount
-					successiveStillRounds = 0
-				} else {
-					if successiveStillRounds > 1 {
-						log.Printf("LeakRoutines commit over.")
-						break
-					}
-					successiveStillRounds++
-				}
-				rounds++
-				time.Sleep(5 * time.Second)
-			}
-			return
+			bucketsTomb.Kill(nil)
```

Comment:
We need to ensure that we are not losing overflows being sent by buckets when doing this. Are we sure this can't lead to losing overflows from bucket that have overflowed but are waiting to be able to send their overflow into the appropriate chan ?

Reply:
* I tried creating 5k+ overflows from special crafted scenarios (trigger + 1 capacity bucket ). We are not loosing any decisions.
* reload seems to happen fine